frame/staking/src/lib.rs

```diff
@@ -3643,8 +2800,9 @@ where
 	}
 
 	fn can_report() -> bool {
-		// TWO_PHASE_NOTE: we can get rid of this API
-		Self::era_election_status().is_closed()
+		// TODO: follow up or a PR on top of this: remove this API now. No need for it anymore.
+		// Needs some small migrations in offences as well, so maybe do it separate.
+		true
```

Comment:
Worth creating an issue now to track the followup?

Reply:
https://github.com/paritytech/substrate/issues/8343